assets/javascripts/job_templates.js

```diff
@@ -260,7 +260,10 @@ function buildMediumGroup(group, media) {
       select.attr('id', group + '-' + arch + '-' + test);
       select.attr('data-product-id', archs[arch]['_id']);
       select.addClass('chosen-select');
-      if (archs.hasOwnProperty(arch) && archs[arch].hasOwnProperty(test)) {
+      if (
```

Comment:
Has it applied these changes programmatically?

Reply:
No, i had to fix those manually. But it formatted the code afterwards with prettier rules.

Reply:
VSCode does make fixing them very easy though. The popup dialog contains a link to the web site that explains why the rule exists and how to fix the violation.
<img width="1310" alt="vscode" src="https://user-images.githubusercontent.com/30094/127513768-4798457f-c585-4d4c-9dd2-928996af95f9.png">

Reply:
I think most of these rules are too tricky for ESLint to fix on its own.

Reply:
Ok, I suppose that would be too much to ask for :-)